build/networking/Dockerfile.ubuntu

```diff
@@ -85,4 +86,4 @@ COPY ./examples /root/examples
 COPY ./start_p4ovs.sh /root/start_p4ovs.sh
 COPY ./run_ovs_cmds /root/run_ovs_cmds
 RUN  /root/start_p4ovs.sh /root && \
-     /root/scripts/run_cleanup.sh /root ${KEEP_SOURCE_CODE}
+     root/scripts/run_cleanup.sh /root ${KEEP_SOURCE_CODE}
```

Comment:
You need a `/` in front of root here.

Reply:
Good catch, this was mistakenly removed. Fixed now